docs/contributing/developing.mdx

```diff
@@ -16,17 +16,48 @@ cd infisical
 
 ## Set up environment variables
 
-Tweak the `.env` according to your preferences. Refer to the available [environment variables](/self-hosting/configuration/envars).
+Before running the docker-compose we have to generate the .env file with the environment variables, you can create your own file or start with the
+`.env.example` as an example guide.
+
+Mandatory variables in the `.env` file:
+
+1. Keys and JWT variables
+
+![image](https://user-images.githubusercontent.com/118568289/206791534-9c9d1431-e83d-49c0-8a54-b373ed0df820.png)
+
+The `.env.example` has these variables empty, you can self generate the `JWT and ENCRYPTION_KEY` with this [32-byte random hex strings generator](https://www.browserling.com/tools/random-hex).
+
+For the `PRIVATE_KEY and PUBLIC_KEY` you can use the ones shown in the screenshot:
 
-```bash
-cp .env.example .env
 ```
+PRIVATE_KEY='oGVv5rThrpZ7WLgQW27chY1cXngr4wLQIZnGfSKgHPk='
+PUBLIC_KEY='ldr6JaC7AY+tun3omGLdE4SWpkJbtVBOI54KfUP53Xc='
+```
+
+2. Mongo variables and site URL
+
+![image](https://user-images.githubusercontent.com/118568289/206792171-3376e3c6-c3ac-4d5d-8776-d78ee089b520.png)
+
+These variables are used to connect the MongoDB and set the URL for the localhost.
```

Comment:
Something to also add here is that `MONGO_USERNAME` and `MONGO_PASSWORD` should appear in `MONGO_URL` after `//` in the form of `MONGO_USERNAME:MONGO_PASSWORD`. The recommendation is to just use `root:example` for development purposes though.

@adrianmarinwork do you think this makes sense to be added here?

Reply:
Yeah, it could be a good thing to pinpoint in the case that the user uses his own username and password.

Reply:
Done!

Reply:
This is perfect!